### ISSUE TITLE
I want to fine tune Lora-FLUX.2-klein-base-4B, but it's still downloading Mistral-3 text encoder from mistralai/Mistral-Small-3.1-24B-Instruct-2503...

### ISSUE THREAD

**Issue by T-1009** [bghira/SimpleTuner#2643]
{
  "data_backend_config": "config/multidatabackend.json",
  "model_type": "lora",
  "model_family": "flux2",
  "model_flavour": "klein-4b",
  "pretrained_model_name_or_path": "/home/ubuntu/SAR/LORA/models/FLUX.2-klein-base-4B",
  "seed": 42,
  "output_dir": "/home/ubuntu/SAR/LORA/models/Lora-FLUX.2-klein-base-4B",
  "train_batch_size": 1,
  "gradient_accumulation_steps": 2,
  "gradient_checkpointing": true,
  "mixed_precision": "bf16",
  "optimizer": "adamw_bf16",
  "learning_rate": 1e-5,
  "lr_scheduler": "cosine",
  "max_train_steps": 10000,
  "num_train_epochs": 0,
  "checkpoint_step_interval": 1000,
  "flux_lora_target": "all" ,
  "validation_seed": 42,
  "validation_step_interval": 200,
  "validation_resolution": "256x256",
  "validation_num_inference_steps": 30,

  "lora_rank": 16
}

here is my cofig,

**Comment by bghira** [comment]
`"pretrained_text_encoder_model_name_or_path": "/home/ubuntu/SAR/LORA/models/Lora-FLUX.2-klein-base-4B/text_encoder"`

should work:


```py
        mistral_path = getattr(self.config, "pretrained_text_encoder_model_name_or_path", None) or MISTRAL_PATH
        mistral_revision = getattr(self.config, "text_encoder_revision", None) or getattr(self.config, "revision", None)

        logger.info(f"Loading Mistral-3 text encoder from {mistral_path}...")

```

**Comment by bghira** [comment]
im not sure why 4b klein would be using the 24b mistral model, can you share logs from the startup
